src/scss/components/_inputs.scss

```diff
@@ -144,6 +144,29 @@ $input-select-margin-right: 1;
   }
 }
 
+.usa-checkbox__input--tile + .usa-checkbox__label--illustrated,
+.usa-radio__input--tile + .usa-radio__label--illustrated {
+  @include at-media('tablet') {
+    display: flex;
+  }
+
+  .usa-radio__image,
+  .usa-checkbox__image {
+    width: 1rem;
+    float: left;
+    margin-right: 0.5rem;
+    @include at-media('tablet') {
+      width: 2rem;
+    }
+  }
+  .usa-radio__label--text,
+  .usa-checkbox__label--text {
+    @include at-media('tablet') {
+      flex: 1;
+    }
+  }
```

Comment:
What are these styles for?

Reply:
This is to keep the sizing of the image when there is a lot of text in the label. I found that if there are multiple lines for either the title or the description text, the image would shrink accordingly.